docs/en/status-communities/set-up-your-community-permissions.md

```diff
@@ -0,0 +1,66 @@
+---
+id: 355
+revision: 0
+language: en
+hide:
+  - navigation
+---
+
+# Set up your community permissions
+
+!!! note ""
+    Currently, you can only configure community permissions using Status desktop.
+
+In Status Communities, owners can configure community permissions using tokens. Tokens are digital assets in a blockchain (such as Ethereum), including virtual coins, collectibles and domain names.
+
+When configuring permissions based on tokens, certain actions within a specific community are only available to users who own a particular token. For instance, owners can [restrict access to their community][set-up-private-and-public-communities] or [delegate admin functions][co-manage-your-status-community] to token holders.
+
+!!! note
+    In Status, your tokens are securely stored in your [Status Wallet][status-wallet-your-quick-start-guide].
+
+## What to expect
+
+- Using token-based permissions, community owners can restrict community access and actions to specific people based on token requirements.
+- Only the community owner can add, edit or delete community-level permissions.
+- You configure community permissions in the same way you [configure channel permissions][configure-channel-permissions]. The only difference is the permissions you grant, either community-level (Become admin or Become member) or channel-level (Moderate, View and post or View only).
+- Use the Become member permission to [restrict access to your community][set-up-private-and-public-communities]. Use the Become admin permission to [delegate admin functions][co-manage-your-status-community] to someone.
```

Comment:
'Become member' is `Become member` in other parts of the article, so I think both this and 'Become admin' should be in `<code>`?

Reply:
I don't think there shouldn't be any special format for that one, so I'm removing the `<code>` format from Become member. Thanks.